### PR TITLE
network(dm): meter network usage from upstream & downstream

### LINKED ISSUE #7933: DM: network metering from dumpling and lightning

### PR DESCRIPTION
**Pull request by buchuitoudegou**
### What problem does this PR solve?

Issue Number: close #7933

### What is changed and how it works?
- separate network metering from upstream & downstream

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: dm/syncer/syncer.go [hunk 1]
```diff
@@ -2981,7 +2981,7 @@ func (s *Syncer) createDBs(ctx context.Context) error {
 	var err error
 	dbCfg := s.cfg.From
 	dbCfg.RawDBCfg = dbconfig.DefaultRawDBConfig().SetReadTimeout(maxDMLConnectionTimeout)
-	fromDB, fromConns, err := dbconn.CreateConns(s.tctx, s.cfg, conn.UpstreamDBConfig(&dbCfg), 1)
+	fromDB, fromConns, err := dbconn.CreateConns(s.tctx, s.cfg, conn.UpstreamDBConfig(&dbCfg), 1, s.cfg.DumpIOTotalBytes, s.cfg.DumpUUID)
```
Comment (D3Hunter) [inline dm/syncer/syncer.go:2984]: should we name it as `SourceIOTotalBytes` if it accounts syncer bandwidth

**Review by D3Hunter — COMMENTED** [review]
rest lgtm

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**